setup.py

```diff
@@ -26,6 +25,10 @@ def build_libcaption_static_lib(self):
         def call(cmd):
             subprocess.check_call(cmd.split(' '), cwd=abs_path)
 
+        # comment logging on stderr
+        call('sed -i /^\s*fprintf(stderr.*;$/s|^|//|;/^\s*fprintf(stderr/,/);$/s|^|//| %s' % os.path.join(abs_path,'src/cea708.c'))
+        call('sed -i /^\s*fprintf(stderr.*;$/s|^|//|;/^\s*fprintf(stderr/,/);$/s|^|//| %s' % os.path.join(abs_path,'src/eia608.c'))
+        call('sed -i /^\s*fprintf(stderr.*;$/s|^|//|;/^\s*fprintf(stderr/,/);$/s|^|//| %s' % os.path.join(abs_path,'src/mpeg.c'))
```

Comment:
[OPT] I think you can create a function which get's the path parameter, since you're doing 3 times the same sed.